makefiles/tor.mk

```diff
@@ -28,29 +28,40 @@ tor: tor-setup libevent openssl xz zstd libscrypt
 		CC=$(CC) \
 		CPP="$(CXX)" \
 		CFLAGS="$(CFLAGS)" \
-		LFLAGS2="$(CFLAGS)"
+		LDFLAGS="$(LDFLAGS)"
 	+$(MAKE) -C $(BUILD_WORK)/tor install \
 		DESTDIR="$(BUILD_STAGE)/tor"
 	mkdir -p $(BUILD_STAGE)/tor/$(MEMO_PREFIX){/Library/LaunchDaemons,$(MEMO_SUB_PREFIX)/libexec}
-	cp -a $(BUILD_INFO)/org.torproject.tor.plist $(BUILD_STAGE)/tor/$(MEMO_PREFIX)/Library/LaunchDaemons
-	cp -a $(BUILD_INFO)/tor-wrapper $(BUILD_STAGE)/tor/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/libexec
+	cp -a $(BUILD_MISC)/tor/org.torproject.tor.plist $(BUILD_STAGE)/tor/$(MEMO_PREFIX)/Library/LaunchDaemons
+	$(SED) -i -e 's|@MEMO_PREFIX@|$(MEMO_PREFIX)|g' -e 's|@MEMO_SUB_PREFIX@|$(MEMO_SUB_PREFIX)|g' \
+		$(BUILD_STAGE)/tor/$(MEMO_PREFIX)/Library/LaunchDaemons/org.torproject.tor.plist
+	cp -a $(BUILD_MISC)/tor/tor-wrapper $(BUILD_STAGE)/tor/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/libexec
+	$(SED) -i -e 's|@MEMO_PREFIX@|$(MEMO_PREFIX)|g' -e 's|@MEMO_SUB_PREFIX@|$(MEMO_SUB_PREFIX)|g' \
+		$(BUILD_STAGE)/tor/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/libexec/tor-wrapper
 	touch $(BUILD_WORK)/tor/.build_complete
 endif
 
 tor-package: tor-stage
 	# tor.mk Package Structure
-	rm -rf $(BUILD_DIST)/tor
-
+	rm -rf $(BUILD_DIST)/tor{,-geoipdb}
+	mkdir -p $(BUILD_DIST)/tor-geoipdb/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share
+	
 	# tor.mk Prep tor
 	cp -a $(BUILD_STAGE)/tor $(BUILD_DIST)
-
+	rm -rf $(BUILD_DIST)/tor/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/tor
```

Comment:
Line 51 isn't needed.

Reply:
It is needed, otherwise `tor` and `tor-geoipdb` would conflict.

Reply:
Only one of the package should have that share folder, then. Otherwise, move all of the files generated by both packages in one package and package it that way.

Reply:
Yes, only `tor-geoipdb` has `share/tor`.